.circleci/config.yml

```diff
@@ -35,7 +35,7 @@ install_cli_from_local_registry: &install_cli_from_local_registry
     startLocalRegistry "$(pwd)/.circleci/verdaccio.yaml"
     setNpmRegistryUrlToLocal
     changeNpmGlobalPath
-    npm install -g @aws-amplify/cli-internal@11.0.3
+    npm install -g @aws-amplify/cli-internal@12.0.0-rc.0d6bca41ff.0
```

Comment:
Do we need to change the version in setup-dev of `package.json` as well?

Reply:
That's right, updated now. It got reverted when I tested build locally with setup-dev command. Good catch.